extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -81,6 +81,33 @@
           "type": "string",
           "description": "Domain of the recipient's email address."
         },
+        "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/attachmentCount": {
+          "title": "Attachment Count",
+          "type": "integer",
+          "description": "Number of attachments in the email message.",
+          "minimum": 0
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/totalAttachmentSize": {
+          "title": "Total Attachment Size",
+          "type": "integer",
+          "description": "Total size of all attachments in bytes."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/attachmentTypes": {
```

Comment:
I am curious if the attachment types and attachementName array sizes would be different; for example there could be multiple attachments of type `image/jpeg` with different names.